bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrPlugin.kt

```diff
@@ -46,6 +68,16 @@ internal class AnrPlugin : Plugin {
         }
 
         if (loaded) {
+            @Suppress("UNCHECKED_CAST")
+            val clz = Class.forName("com.bugsnag.android.NdkPlugin") as Class<Plugin>
```

Comment:
This needs to catch a `ClassNotFoundException` if the `NdkPlugin` is not present - otherwise the ANR plugin will fail to load